examples/write.rs

```diff
@@ -1,11 +1,11 @@
 use std::fs::OpenOptions;
 use std::io::{self, prelude::*};
 
-use fatfs::{FileSystem, FsOptions};
+use fatfs::{FileSystem, FsOptions, StdIoWrapper};
 use fscommon::BufStream;
 
 fn main() -> io::Result<()> {
-    let img_file = match OpenOptions::new().read(true).write(true).open("fat.img") {
+    let img_file = match OpenOptions::new().create(true).read(true).write(true).open("fat.img") {
```

Comment:
`create(true)` does not make sense for me here. You are not formatting the filesystem. If `fat.img` does not exist it will fail trying to read BPB.